python/cudf/cudf/core/column/column.py

```diff
@@ -89,6 +88,11 @@
 )
 from cudf.utils.utils import _array_ufunc, mask_dtype
 
+if TYPE_CHECKING:
+    import builtins
```

Comment:
This should not be necessary? Try replacing `builtins.slice` with just `slice` where it is used.

Reply:
Looks like this one is needed because mypy thinks `: slice` in this file is the `ColumnBase.slice` function defined in this file (I have no idea why), so I'll leave this as is

Reply:
Ughhhh. Now that you say that, I think I have commented on this before.